test/Dialect/SV/hw-memsim.mlir

```diff
@@ -66,64 +66,6 @@ hw.module @WriteOrderedDifferentClock(%clock: i1, %clock2: i1, %w0_addr: i4, %w0
   hw.output
 }
 
-hw.module.generated @FIRRTLMem_1_1_1_16_10_0_1_0_0, @FIRRTLMem(%ro_addr_0: i4, %ro_en_0: i1, %ro_clock_0: i1,%rw_addr_0: i4, %rw_en_0: i1,  %rw_clock_0: i1, %rw_wmode_0: i1, %rw_wdata_0: i16,  %wo_addr_0: i4, %wo_en_0: i1, %wo_clock_0: i1, %wo_data_0: i16) -> (ro_data_0: i16, rw_rdata_0: i16) attributes {depth = 10 : i64, numReadPorts = 1 : ui32, numReadWritePorts = 1 : ui32, numWritePorts = 1 : ui32, readLatency = 0 : ui32, readUnderWrite = 0 : ui32, width = 16 : ui32, writeClockIDs = [], writeLatency = 1 : ui32, writeUnderWrite = 0 : i32}
-
-//CHECK-LABEL: @FIRRTLMem_1_1_1_16_10_0_1_0_0
```

Comment:
In the previous example, the test which did not use any of the additional registers was first, so there were no leftover registers to generate code for. However, when moved to the end, this test was failing before the fix.

Reply:
Got it. Given that that is what is going on here, it would be simpler to add a test at the end that is a copy of this test (but does not check anything other than the `LABEL`) with a comment explaining why this is there and a link to the issue/PR.

Reply:
done

Reply:
Nice! Much cleaner (to me at least).